spacex/api.py

```diff
@@ -0,0 +1,47 @@
+import requests
+from config import SPACEX_API_URL
+from dataclasses import dataclass
+from requests.exceptions import HTTPError, ConnectionError
+
+
+@dataclass
+class Flight:
+    flight_number: int
+    mission_name: str
+    rocket_id: str
+    rocket_name: str
+    launch_date_utc: str
+    video_link: str
+    data: dict
+
+
+class SpaceXApi:
+    @staticmethod
+    def request_data():
+        try:
+            response = requests.get(SPACEX_API_URL)
+            response.raise_for_status()
+        except HTTPError:
```

Comment:
This can be combined into one except statement.
```suggestion
        except (HTTPError, ConnectionError, TimeoutError):
```